core/src/main/java/apoc/trigger/TriggerNewProcedures.java

```diff
@@ -0,0 +1,129 @@
+            return Stream.of(
+                    TriggerInfo.from(removed, false),
+                    new TriggerInfo( name, statement, selector, params, true, false));
+        }
+        return Stream.of(new TriggerInfo( name, statement, selector, params, true, false));
+    }
+
+
+    // TODO - change with @SystemOnlyProcedure
+    @SystemProcedure
+    @Admin
+    @Procedure(mode = Mode.WRITE)
+    @Description("CALL apoc.trigger.drop(databaseName, name) | eventually removes an existing trigger, returns the trigger's information")
+    public Stream<TriggerInfo> drop(@Name("databaseName") String databaseName, @Name("name")String name) {
+        checkInSystemWriter();
+        Map<String, Object> removed = TriggerHandlerNewProcedures.drop(databaseName, name);
+        if (!removed.containsKey(SystemPropertyKeys.statement.name())) {
+            return Stream.of(new TriggerInfo(name, null, null, false, false));
+        }
+        return Stream.of(TriggerInfo.from(removed, false));
+    }
+    
+    
+    // TODO - change with @SystemOnlyProcedure
+    @SystemProcedure
+    @Admin
+    @Procedure(mode = Mode.WRITE)
+    @Description("CALL apoc.trigger.dropAll(databaseName) | eventually removes all previously added trigger, returns triggers' information")
+    public Stream<TriggerInfo> dropAll(@Name("databaseName") String databaseName) {
+        checkInSystemWriter();
+        Map<String, Object> removed = TriggerHandlerNewProcedures.dropAll(databaseName);
+        return removed.entrySet().stream().map(TriggerInfo::entryToTriggerInfo);
+    }
+
+    // TODO - change with @SystemOnlyProcedure
+    @SystemProcedure
+    @Admin
+    @Procedure(mode = Mode.WRITE)
+    @Description("CALL apoc.trigger.stop(databaseName, name) | eventually pauses the trigger")
+    public Stream<TriggerInfo> stop(@Name("databaseName") String databaseName, @Name("name")String name) {
+        checkInSystemWriter();
+        Map<String, Object> paused = TriggerHandlerNewProcedures.updatePaused(databaseName, name, true);
+
+        return Stream.of(TriggerInfo.from(paused,true));
+    }
+
+    // TODO - change with @SystemOnlyProcedure
+    @SystemProcedure
+    @Admin
+    @Procedure(mode = Mode.WRITE)
+    @Description("CALL apoc.trigger.start(databaseName, name) | eventually unpauses the paused trigger")
+    public Stream<TriggerInfo> start(@Name("databaseName") String databaseName, @Name("name")String name) {
+        checkInSystemWriter();
+        Map<String, Object> resume = TriggerHandlerNewProcedures.updatePaused(databaseName, name, false);
+
+        return Stream.of(TriggerInfo.from(resume, true));
+    }
+
+    // TODO - change with @SystemOnlyProcedure
+    @SystemProcedure
+    @Procedure(mode = Mode.READ)
+    @Description("CALL apoc.trigger.show(databaseName) | it lists all eventually installed triggers for a database")
+    public Stream<TriggerInfo> show(@Name("databaseName") String databaseName) {
```

Comment:
It seems like we added the @Admin annotation to the apoc.trigger.list procedure in 4.4 but not the apoc.trigger.show procedure. I think that's a little inconsistent and should be changed in both 4.4 and 5.x so that all trigger procedures have the @Admin procedure since that seems to be the direction we decided to go.

Reply:
My bad :( 
In 4.4 is present, for some reason I didn't put it here as well. Added